LEGO1/lego/legoomni/include/legoracecar.h

```diff
@@ -31,8 +39,19 @@ class LegoRaceCar : public LegoCarRaceActor {
 	void VTable0x98() override;                        // vtable+0x98
 	void VTable0x9c() override;                        // vtable+0x9c
 
+	virtual void FUN_10012ea0(float p_worldSpeed);
+	virtual void FUN_10012ff0(float);
+	virtual MxBool FUN_10013130(float);
+
 	// SYNTHETIC: LEGO1 0x10014230
 	// LegoRaceCar::`scalar deleting destructor'
+	byte m_unk0x54;           // 0x54
```

Comment:
Is there a particluar reason to use `byte`? We use `undefined` for an undefined byte.